orangecontrib/text/widgets/owscraper.py

```diff
@@ -0,0 +1,160 @@
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from Orange.data import Table, StringVariable
+from newspaper import Article
+import tldextract
+from datetime import datetime
+from PyQt4 import QtCore
+
+ARTICLE_TEXT_FIELDS=["Article","Author","Date","Location","Title","URL"]
```

Comment:
okay :+1: 

There is a small error

line 147, in set_data
    self.generate_table(data)
  File "/home/cheral/orange3-text/orangecontrib/text/widgets/owscraper.py", line 126, in generate_table
    info=get_info(url)
NameError: name 'get_info' is not defined

any changes suggested?